packages/components/pie-checkbox/src/index.ts

```diff
@@ -87,7 +87,7 @@ export class PieCheckbox extends RtlMixin(LitElement) implements CheckboxProps {
                 name=${ifDefined(name)}
                 ?disabled=${disabled}
                 ?required=${required}
-                .indeterminate=${indeterminate}
+                .indeterminate=${indeterminate || defaultProps.indeterminate}
```

Comment:
I'm unsure if this is necessary, given we also set the default on L46

Reply:
And the same comment for the spinner component
https://github.com/justeattakeaway/pie/pull/1451/files#diff-766e0e088420ac03dd430a518fb113540ebc722ea339697d32c2ebb342015bddR44-R45